pkg/util/status/clusterversionoperator_status.go

```diff
@@ -10,6 +10,8 @@ import (
 const OperatorFailing configv1.ClusterStatusConditionType = "Failing"
```

Comment:
I think move the comment to here ^ and also make a ticket in vsts with the same info

Reply:
Done https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/8024644